conans/model/conan_file.py

```diff
@@ -285,3 +289,28 @@ def __repr__(self):
             return "%s/%s@PROJECT" % (self.name, self.version)
         else:
             return "PROJECT"
+
+
+def parse_editable_cpp_info(content, base_path, settings=None, options=None):
```

Comment:
This function might belong to the `installer`. It is not used in another place.

Reply:
Or probably even better to create a model (maybe just a module) for the ` .conan_package_layout ` and move there the parsing/evaluating?

Reply:
Agree, not here. Sounds like EditableCppInfo might deserve its own model